src/transformers/models/ibert/utils/quant_modules.py

```diff
@@ -0,0 +1,538 @@
+import torch
+import time
+import math
+import numpy as np
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.multiprocessing as mp
+from torch.nn import Linear as _linear
+from torch.nn import Embedding as _Embedding
+from torch.nn import Module, Parameter
+from .quant_utils import *
```

Comment:
not a big fan of the star import, I think it would be nicer if we import all the functionality that we need